devimint/src/tests.rs

```diff
@@ -136,17 +137,11 @@ pub async fn latency_tests(
     let initial_balance_sats = 100_000_000;
     fed.pegin_client(initial_balance_sats, &client).await?;
 
-    // LN operations take longer, we need less iterations
-    let iterations = 20;
-
     let cln_gw_id = gw_cln.gateway_id().await?;
 
     match r#type {
         LatencyTest::Reissue => {
             info!("Testing latency of reissue");
-            // On AlephBFT session times may lead to high latencies, so we need to run it
```

Comment:
Not worth waiting for, plus probably requires like hundreds of iterations to really hit it.